src/javascript/app_2/Modules/Trading/Components/Form/TradeParams/amount.jsx

```diff
@@ -44,16 +45,15 @@ const Amount = ({
     return (
         <Fieldset>
             <div className={amount_container_class}>
-                <Dropdown
-                    is_alignment_left
-                    is_nativepicker={is_nativepicker}
-                    list={basis_list}
+                <ButtonToggleMenu
```

Comment:
Possible to use the `ToggleButtonGroup` component for the button menu? (will replace the ButtonToggleMenu in Duration with this one as well)

Reply:
I will do it in another card whenever the design of ToggleButtonGroup is fixed, currently it has some issues